### ISSUE TITLE
Enforce the entire application to only load over HTTPS

### ISSUE THREAD

**Issue by thescientist13** [ProvidenceGeeks/website-frontend#130]
## Type of Change
- [ ] New Feature Request
- [x] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary

As part of the Lighthouse audit, the application should run entirely over HTTPS, including forwarding any HTTP traffic to HTTPS traffic.

**Comment by thescientist13** [comment]
Making a cURL request confirms that the redirect is in place in both environments

**STAGE**
```shell
$ curl -I http://stage.pvdgeeks.org
HTTP/1.1 301 Moved Permanently
Server: CloudFront
Date: Tue, 09 Jan 2018 16:56:06 GMT
Content-Type: text/html
Content-Length: 183
Connection: keep-alive
Location: https://stage.pvdgeeks.org/
X-Cache: Redirect from cloudfront
Via: 1.1 f83a7605cde7ddbe4c1e27d757caa746.cloudfront.net (CloudFront)
X-Amz-Cf-Id: RYE4T5oK-yhnpOrZBPH-YFlrxubC-M1eF8byV-iRZJQ44KamabQEvA==
```

**PROD**
```shell
$ curl -I http://www.pvdgeeks.org
HTTP/1.1 301 Moved Permanently
Server: CloudFront
Date: Tue, 09 Jan 2018 16:57:36 GMT
Content-Type: text/html
Content-Length: 183
Connection: keep-alive
Location: https://www.pvdgeeks.org/
X-Cache: Redirect from cloudfront
Via: 1.1 d2732303fd1054eafb760187913561e3.cloudfront.net (CloudFront)
X-Amz-Cf-Id: _8zehIz1JQ1nC6O4FyrLESC8o3OzUkSy-TO44G-qXaWQCFPXoxKSlg==
```

**Comment by thescientist13** [comment]
This should fix #90 🤞
